manager/testdata/notebook/README.md

```diff
@@ -4,12 +4,13 @@
 are configured to use TLS:
 
 - OPA connector
-- katalog connector
+- Openmetadata connector
+- Openmetadata server
 - OPA server
 - Fybrik manager
 - Vault
 
-All servers above use mutual TLS.
+All servers above except Openmetadata server use mutual TLS.
```

Comment:
it changes the TLS tests to use openmetadata as catalog instead of katalog. openmetadata server is not configured to work with tls, only openmetadata connector.